protos/taikai/v1/types.proto

```diff
@@ -55,6 +55,14 @@ message User {
 	optional string zip = 13 [json_name = "zip"];
 	// @gotags: fake:"{jobtitle}"
 	optional string job_title = 17 [json_name = "job_title"];
+	// @gotags: fake:"skip"
+	optional string org_id = 18 [json_name = "org_id"];
```

Comment:
So a User can only be part of one org?

Reply:
I don't know the use case for more than one org at this point in time. I am sure we should care about at some point